data/containers/autoyast_containers.xml.ep

```diff
@@ -79,6 +77,26 @@
     <keep_install_network config:type="boolean">true</keep_install_network>
     % }
   </networking>
+  % if ($check_var->('VERSION', '15-SP4')) {
```

Comment:
For some reason, in 15-SP4 we need to have this enabled, otherwise s390x can't login to the VM afterwards... 
e.g. without this block: https://openqa.suse.de/tests/7287263#step/grub_set_bootargs/1